### DISCUSSION TITLE
Dynamic redirects currently in vercel.json

### DISCUSSION THREAD

**Discussion by mattpilott** [vercel/community#2078] [Help] [answered]
### Summary

Is there a way to have an equivalent to the redirect functionality in vercel.json but where the redirects come from an api/cms ?

I'm using sveltekit

**Comment by amyegan** [comment, marked as answer]
Hi @mattpilott! You can throw a [`redirect`](https://kit.svelte.dev/docs/modules#sveltejs-kit-redirect) with SvelteKit ([example](https://learn.svelte.dev/tutorial/redirects))

**Reply by mattpilott** [reply]
You're right, i'm aware of `redirect` however my understanding is that doing it in a similar way to vercel.json would mean the redirect happens without touching or firing up my application code, is that right?

**Reply by amyegan** [reply]
In that case, I think [Edge Middleware redirects](https://vercel.com/docs/concepts/edge-network/redirects#edge-middleware) are what you're looking for. You can find more info in the recently updated Redirects documentation at https://vercel.com/docs/concepts/edge-network/redirects
